### ISSUE TITLE
Criar funções para a validação de campos de cadastro

### ISSUE THREAD

**Issue by jorgeprj** [Lvght/e-andifes#88]
Acho que faria sentido criar funções que validam CPF, Endere;o, Telefone e qualquer outro campo que deve ser preenchido pelo usuário para qualquer cadastro

**Comment by JonathanBDias** [comment]
Concordo, visto que se utiliza esses campos em vários locais e eles devem ser validados, já que estamos trabalhando com um sistema complexo e dados reais.
